File: Python/Remove_Item.py
def remove(self, item):
    if self.is_empty():
        print("Queue is empty. Cannot remove.")
        return
    i = self.front
    while i <= self.rear:
        if self.array[i] == item:
            for j in range(i, self.rear):
                self.array[j] = self.array[j + 1]
            self.rear -= 1
            return
        i += 1
    print("Item not found in the queue.")

File: C/Remove_Item.c
void remove(struct Queue *queue, int item) {
    if (isEmpty(queue)) {
        printf("Queue is empty. Cannot remove.\n");
        return;
    }
    int i, j;
    for (i = queue->front, j = 0; i <= queue->rear; i++, j++) {
        if (queue->array[i] == item) {
            for (int k = i; k < queue->rear; k++) {
                queue->array[k] = queue->array[k + 1];
            }
            queue->rear--;
            return;
        }
    }
    printf("Item not found in the queue.\n");
}
